test/azure/terraform_azure_datafactory_example_test.go

```diff
@@ -0,0 +1,49 @@
+package test
```

Comment:
can you add the build tags for ci test execution?
```suggestion
//go:build azure
// +build azure

// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
// CircleCI.
```

Reply:
Thanks Hadwa added,also updated the CI pipeline run above in description for datafactory and synapse passed

https://github.com/sweanan/terratest/actions/runs/5327539258/jobs/9651062907

--- PASS: TestTerraformAzureSynapseExample (963.67s)
--- PASS: TestTerraformAzureDataFactoryExample (65.98s)